tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -0,0 +1,136 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
```

Comment:
I'd like to see a test (can be one of the existing ones) that goes through the `CoordinateMap` class to make sure the implementation works correctly with that.